src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -370,7 +370,9 @@ public function getModelData($form_data, $isMultiShopContext = false)
             $form_data['combinations'][$k]['attribute_unity'] = abs(
                 $this->floatParser->fromString($combination['attribute_unity'])
             );
-            $form_data['combinations'][$k]['attribute_quantity'] = $this->floatParser->fromString($combination['attribute_quantity']);
+            if (ConfigurationLegacy::get('PS_STOCK_MANAGEMENT')) {
```

Comment:
This is now forbidden. It already exists in the file twice and is tolerated but you're adding a new one that triggers PHPStan ans you can read more easily in the [action logs](https://github.com/PrestaShop/PrestaShop/actions/runs/4186592753/jobs/7255291051#step:7:59). I am not sure what is the new way to do so tho

Reply:
I did a refactor based on the Symfony service `prestashop.adapter.legacy.configuration` (see https://devdocs.prestashop-project.org/8/development/multistore/shops-configuration/#usage).

`ConfigurationLegacy` is not used in this classe anymore 👍

Reply:
Awesome ! Can you then remove in `phpstan-baseline.neon` the lines 1707 to 1711 ?
```

		-
			message: "#^Namespace Configuration is forbidden, No legacy calls inside the prestashop bundle\\. Please create an interface and an adapter if you need to\\.$#"
			count: 2
			path: src/PrestaShopBundle/Model/Product/AdminModelAdapter.php
```
No longer needed as you refacto'd the whole file

Reply:
It's done, thanks for your help